### PR TITLE
Add session inactive timeout configuration (task #1734)

### PR DESCRIPTION
**Pull request by flangofas**

### CODE REVIEW

**Comment by leonidsl** [comment]
:+1:

File: config/app.php [hunk 1]
```diff
@@ -21,6 +21,7 @@
 $sessionCookieSecure = (bool)env('APP_SESSION_SECURE_COOKIE');
 $sessionCookieSecure = $https ?: $sessionCookieSecure;
 $cookieHttpOnly = (bool)env('APP_SESSION_COOKIE_HTTP_ONLY');
+$sessionTimeout = env('APP_SESSION_TIMEOUT');
```
Comment (mamchenkov) [inline config/app.php]: @flangofas `$sessionTimeout = (int)env('APP_SESSION_TIMEOUT');` seems appropriate ;)

Comment (flangofas) [inline config/app.php]: @mamchenkov Empty strings will be 0 then.

Comment (mamchenkov) [inline config/app.php]: @flangofas yup, and 0 disables session timeout, if I remember correctly.

Comment (flangofas) [inline config/app.php]: @mamchenkov disable session timeout or default to php.ini?

Comment (mamchenkov) [inline config/app.php]: @flangofas I think it 0 means disable the timeout ... who handles the session - php or cakephp - is a different setting.

**Comment by flangofas** [comment]
@mamchenkov please review.

**Pull request merged**